src/sentry/static/sentry/app/components/pill.jsx

```diff
@@ -29,19 +29,84 @@ class Pill extends React.Component {
   };
 
   render() {
-    const {name, children, className, ...props} = this.props;
+    const {name, children, ...props} = this.props;
     const [extraClass, renderedValue] = this.renderValue();
-
     return (
-      <li className={classNames(className, extraClass)} {...props}>
-        <span className="key">{name}</span>
-        <span className="value">
+      <StyledPill {...props}>
+        <PillName>{name}</PillName>
+        <PillValue type={extraClass}>
           {renderedValue}
           {children}
-        </span>
-      </li>
+        </PillValue>
+      </StyledPill>
     );
   }
 }
 
+const StyledPill = styled('li')`
+  white-space: nowrap;
+  margin: 0 10px 10px 0;
+  display: flex;
+  border: 1px solid ${p => p.theme.gray1};
+  border-radius: 3px;
```

Comment:
we have a default border radius in theme if you wanna use it, tho sometimes on these smaller boxes it looks bad.

Reply:
@Chrissy I used the `${p => p.theme.button.borderRadius}` which is 3px